script/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o xtrace
+
+export BIND_ADDR
+BIND_ADDR=$(hostname --ip-address)
```

Comment:
FYI, this is done because same-line export and assign would mean errexit probably doesn't do what you want; if hostname failed, export would still succeed.